### ISSUE TITLE
oj.getClassList: Array.append should be replaced by .push.

### ISSUE THREAD

**Issue by IngwiePhoenix** [musictheory/oj#50]
I just wanted to see what I can do with `oj.getClassList`. Turns out it has a bug.

```
> console.log(oj.getClassList.toString())
function getClassList()
{
    var results = [ ];

    for (var key in _classNameToClassMap) { if (hop(_classNameToClassMap, key)) {
        results.append(_classNameToClassMap[key]);
        ^^^^^^^^^^^^^^ Should use .push(...)
    }}

    return results;
}
```

**Comment by IngwiePhoenix** [comment]
This issue still exists and I wonder why... o.o
Could you fix it? I am currently needing to `Array.prototype.append = Array.prototype.push`...which is pretty nasty. :P

**Comment by iccir** [comment]
Sorry about that!  Published to npm as 1.1.3
